src/main/scala/lenthall/spray/WrappedRoute.scala

```diff
@@ -1,31 +0,0 @@
-package lenthall.spray
-
-import spray.routing._
-import spray.routing.directives.PathDirectives
-
-object WrappedRoute {
```

Comment:
Make sure the code is copied cromwell _and_ [thurloe](https://github.com/broadinstitute/thurloe/blob/0089c2646ee79a10f77a2ba1aad592d02220bf84/src/main/scala/thurloe/service/ThurloeServiceActor.scala#L7) too!